packages/backend/server/src/cache/redis.ts

```diff
@@ -0,0 +1,194 @@
+import { Redis } from 'ioredis';
```

Comment:
why dont use keyv's redis adapter? https://github.com/jaredwray/keyv/tree/main/packages/redis

Reply:
it's a pure kv store abstraction, with only `get(k)`, `set(k, v)` interface, and we can't take advantages from redis's original list/hash structrue.